src/content/en/showcase/2017/with.in.md

```diff
@@ -0,0 +1,331 @@
+project_path: /web/_project.yaml
+book_path: /web/showcase/_book.yaml
+
+{# wf_published_on: 2017-02-02T18:00:00.000Z #}
+{# wf_updated_on: 2017-02-02T18:00:00.000Z #}
+{# wf_featured_image: /web/showcase/2017/images/with.in/featured.png #}
+{# wf_featured_snippet: Within is a platform for storytelling in Virtual Reality and is available everywhere VR is. This includes the web. Leveraging WebVR, viewers are able to go to the website, click a link, and immediately watch Within’s films in immersive VR - including high end head-mounted displays. During the development process the team discovered rendering text is difficult in this new environment, and they created an example using shaders to make it a smoother process. #}
+{# wf_tags: webvr,casestudy #}
+
+
+# Rendering Text in WebVR {: .page-title }
+
+**Project:** [https://vr.with.in/](https://vr.with.in/)
+
+**Demo:** [https://vr.with.in/archive/text-sdf-bitmap/](https://vr.with.in/archive/text-sdf-bitmap/)
+
+**Working Files:** [https://vr.with.in/archive/text-sdf-bitmap.zip](https://vr.with.in/archive/text-sdf-bitmap.zip)
+
+<img src="/web/showcase/2017/images/with.in/0.png">
```

Comment:
Please refactor this so it matches others, with intro on left, image floated on the right, and links below in the intro